CODE_STYLE.md

```diff
@@ -11,3 +11,12 @@ MatcherAssert.assertThat(target, new IsEquals<>(expected));
 // don't use
 MatcherAssert.assertThat(target, Matchers.isEquals(expected));
 ```
+
+2. Avoid adding reason to assertions in unit tests:
```

Comment:
@olegmoz I'd say:
Avoid adding reason to assertions in unit tests with **single assertion**:

since assertion message may useful in multi assert unit tests.

Reply:
@g4s8 agree, fixed it